deployment/GI03FLMB_Deploy.csv

```diff
@@ -121,22 +121,22 @@ AR30-03,,AR35-05,,GI03FLMB-RIM01-02-CTDMOG064,5,1,2018-06-11T14:35:00,2019-08-10
+AR30-03,,AR35-05,,GI03FLMB-RIM01-02-CTDMOH071,5,1,2018-06-11T14:35:00,2019-08-10T14:15:00,CGMGI-03FLMB-00005,,CGINS-CTDMOH-13632,59.71427,-39.31329,,1500,2829,
 AR30-03,,AR35-05,,GI03FLMB-RIS01-00-SIOENG000,5,1,2018-06-11T14:35:00,2019-08-10T12:09:00,CGMGI-03FLMB-00005,,CGCON-SIOCON-14871,59.71427,-39.31329,,30,2829,
 AR30-03,,AR35-05,,GI03FLMB-RIS01-03-DOSTAD000,5,1,2018-06-11T14:35:00,2019-08-10T12:09:00,CGMGI-03FLMB-00005,,CGINS-DOSTAD-00394,59.71427,-39.31329,,30,2829,
 AR30-03,,AR35-05,,GI03FLMB-RIS01-04-PHSENF000,5,1,2018-06-11T14:35:00,2019-08-10T12:09:00,CGMGI-03FLMB-00005,,CGINS-PHSENF-P0181,59.71427,-39.31329,,30,2829,
 AR30-03,,AR35-05,,GI03FLMB-RIS01-05-FLORTD000,5,1,2018-06-11T14:35:00,2019-08-10T12:09:00,CGMGI-03FLMB-00005,,CGINS-FLORTD-01320,59.71427,-39.31329,,30,2829,
-AR35-05,,,,GI03FLMB-RI000-00-CTDMOH000,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-13454,59.720515,-39.35474,,1819,2819,
-AR35-05,,,,GI03FLMB-RI000-00-CTDMOH100,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-13457,59.720515,-39.35474,,2719,2819,
-AR35-05,,,,GI03FLMB-RI000-00-CTDMOH400,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-13456,59.720515,-39.35474,,2419,2819,
-AR35-05,,,,GI03FLMB-RI000-00-CTDMOH700,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-13455,59.720515,-39.35474,,2119,2819,
-AR35-05,,,,GI03FLMB-RI000-00-VELPTB000,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-VELPTB-12236,59.720515,-39.35474,,1819,2819,
-AR35-05,,,,GI03FLMB-RI000-00-VELPTB100,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-VELPTB-12673,59.720515,-39.35474,,2719,2819,
-AR35-05,,,,GI03FLMB-RI000-00-VELPTB400,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-VELPTB-12663,59.720515,-39.35474,,2419,2819,
-AR35-05,,,,GI03FLMB-RI000-00-VELPTB700,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-VELPTB-12459,59.720515,-39.35474,,2119,2819,
+AR35-05,,,,GI03FLMB-RI000-00-CTDMOH000,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-13454,59.720515,-39.35474,,1812,2819,
+AR35-05,,,,GI03FLMB-RI000-00-CTDMOH100,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-13457,59.720515,-39.35474,,2712,2819,
+AR35-05,,,,GI03FLMB-RI000-00-CTDMOH400,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-13456,59.720515,-39.35474,,2412,2819,
+AR35-05,,,,GI03FLMB-RI000-00-CTDMOH700,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-CTDMOH-13455,59.720515,-39.35474,,2112,2819,
+AR35-05,,,,GI03FLMB-RI000-00-VELPTB000,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-VELPTB-12236,59.720515,-39.35474,,1812,2819,
+AR35-05,,,,GI03FLMB-RI000-00-VELPTB100,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-VELPTB-12673,59.720515,-39.35474,,2712,2819,
+AR35-05,,,,GI03FLMB-RI000-00-VELPTB400,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-VELPTB-12663,59.720515,-39.35474,,2412,2819,
+AR35-05,,,,GI03FLMB-RI000-00-VELPTB700,6,1,2019-08-08T13:20:00,,CGMGI-03FLMB-00006,,CGINS-VELPTB-12459,59.720515,-39.35474,,2112,2819,

```

Comment:
The OSNAP depths are incorrect (as is the spreadsheet - my bad) they should:
1812 –> 1819
2712 –> 2719
2412 –> 2419
2112 –> 2119

Reply:
corrected